tests/Zend/Date/DateObjectTest.php

```diff
@@ -614,6 +614,20 @@ public function testToStringShouldEqualWithAndWithoutPhpFormat()
             $date->toString(DateTime::ATOM, 'php')
         );
     }
+
+    public function testIncompleteTimeData()
+    {
+        try {
+            $date = new Zend_Date();
+            $date->set(0, Zend_Date::TIMES);
+        } catch (\Exception $e) {
+            $this->fail('This usually stems from not hand iconv_substr propert');
```

Comment:
please use complete English, not geek abbreviations

Reply:
That's not geek, that's me writing stuff half asleep and not double checking before comitting :)

Reply:
In any case, I'm not able to decipher what that error message means, so it needs to be improved.